src/css/information.scss

```diff
@@ -2,15 +2,25 @@
 @import './core/mixins';
 
 #information {
+  &:after {
+    content: '';
+    display: block;
+    clear: both;
+  }
   .quick-id,
   .contact-me {
     padding: 70px 40px;
     min-height: 520px;
+    width: 50%;
+    float: left;
```

Comment:
Don't you want to give it a try with flexbox?
What you are doing is absolutely fine :P